lib/assessment.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.inject { |a, b| a + b }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    enum.each.odd?
```

Comment:
Sorry, that's not right. First, there is a method to see if at least some members of a collection pass a particular test. Second, The syntax is incorrect.